core/tests/protractor/loginFlow.js

```diff
@@ -0,0 +1,122 @@
+    });
+
+  describe('profile dropdown menu', function() {
+    beforeEach(function() {
+      users.login('desktopAndMobileVisitor@loginFlow.com');
+      learnerDashboardPage.get();
+      var profileDropdown = element(by.css(
+        '.protractor-test-profile-dropdown'));
+      waitFor.elementToBeClickable(
+        profileDropdown, 'Could not click profile dropdown');
+      profileDropdown.click();
+    });
+
+    it('should visit the profile page from the profile dropdown menu',
+      function() {
+        var profileLink = element(by.css(
+          '.protractor-test-profile-link'));
+        waitFor.elementToBeClickable(
+          profileLink, 'Could not click on the profile link');
+        profileLink.click();
+        waitFor.pageToFullyLoad();
+        expect(browser.getCurrentUrl()).toEqual('http://localhost:9001/profile/desktopAndMobileVisitor');
+      });
+
+    it('should visit the creator dashboard from the profile dropdown menu',
+      function() {
+        var creatorDashboardLink = element(by.css(
+          '.protractor-test-creator-dashboard-link'));
+        waitFor.elementToBeClickable(
+          creatorDashboardLink,
+          'Could not click on the creator dashboard link');
+        creatorDashboardLink.click();
+        waitFor.pageToFullyLoad();
+        expect(browser.getCurrentUrl()).toEqual('http://localhost:9001/creator_dashboard');
+      });
+
+    it('should visit the learner dashboard from the profile dropdown menu',
+      function() {
+        var learnerDashboardLink = element(by.css(
+          '.protractor-test-learner-dashboard-link'));
+        waitFor.elementToBeClickable(
+          learnerDashboardLink,
+          'Could not click on the learner dashboard link');
+        learnerDashboardLink.click();
+        waitFor.pageToFullyLoad();
+        expect(browser.getCurrentUrl()).toEqual('http://localhost:9001/learner_dashboard');
+      });
+
+    it('should visit the notifications page from the profile dropdown menu',
+      function() {
+        var notificationsDashboardLink = element(by.css(
+          '.protractor-test-notifications-link'));
+        waitFor.elementToBeClickable(
+          notificationsDashboardLink,
+          'Could not click on the notifications dashboard link');
+        notificationsDashboardLink.click();
+        waitFor.pageToFullyLoad();
+        expect(browser.getCurrentUrl()).toEqual('http://localhost:9001/notifications_dashboard');
+      });
+
+    it('should visit the preferences page from the profile dropdown menu',
+      function() {
+        var preferencesLink = element(by.css(
+          '.protractor-test-preferences-link'));
+        waitFor.elementToBeClickable(
+          preferencesLink,
+          'Could not click on the preferences link');
+        preferencesLink.click();
+        waitFor.pageToFullyLoad();
+        expect(browser.getCurrentUrl()).toEqual('http://localhost:9001/preferences');
+      });
+
+    afterEach(function() {
```

Comment:
Given that this is already done on line 118, do you need this block?

Reply:
No, removed. Thanks!